front/src/services/AuthService.js

```diff
@@ -0,0 +1,50 @@
+import axios from 'axios';
+import queryString from 'query-string';
+
+const API_URL = process.env.REACT_APP_SERVER_URL;
+
+const AuthService = {
+  login: async (email, password) => {
+    const optionsToLogin = {
+      url: `${API_URL}/oauth/token`,
```

Comment:
API_URL is in constant file. Please put '/oaut/token' (and every magical URL string) in a dedicated constant file. This will help you gather altogether all urls and therefore will ease maintenance.